python3/cloudwatch/src/lambda_function.py

```diff
@@ -128,6 +121,7 @@ def lambda_handler(event, context):
             raise TypeError("Expected log inside logEvents to be a dict but found another type")
 
         _parse_cloudwatch_log(log, additional_data)
-        shipper.add(log)
+        if not log['message'].startswith('START') and not log['message'].startswith('END') and not log['message'].startswith('REPORT'):
```

Comment:
Why here? Also, let's not use `not` it makes it hard to read the code

Reply:
please review changes on new commit
https://github.com/logzio/logzio_aws_serverless/commit/8b2689012a9bf9f925a2da9f506b514186d2e20d